src/webhooks/pubsub/slackScores.ts

```diff
@@ -0,0 +1,90 @@
+import moment from 'moment-timezone';
+
+import { PRODUCT_OWNERS_INFO, TEAM_OSPO_CHANNEL_ID } from '@/config';
+import { getIssueEventsForTeam } from '@/utils/scores';
+import { GitHubOrg } from '@api/github/org';
+import { bolt } from '@api/slack';
+
+type teamScoreInfo = {
+  team: string;
+  score: number;
+  eventsTriagedOnTime: number;
+  numEvents: number;
+};
+
+const TEAM_COLUMN_WIDTH = 30;
+const SCORE_COLUMN_WIDTH = 15;
+const TEAM_PREFIX = 'team-';
+
+export const triggerSlackScores = async (
+  __org: GitHubOrg,
+  __now: moment.Moment
+) => {
+  const teamScores: teamScoreInfo[] = await Promise.all(
+    Object.keys(PRODUCT_OWNERS_INFO['teams']).map(async (team: string) => {
+      const issueTriageEvents = await getIssueEventsForTeam(team);
+      const triagedEvents = issueTriageEvents.filter(
+        (issue) => issue.is_triaged
+      );
+      const score =
+        issueTriageEvents.length === 0
+          ? -1
```

Comment:
Why is this -1, and not 0? Am I correct in assuming that it's to represent the different between "team had no events" vs "team had events, but did 0 of them". If so, I'd do an optional type instead here, that way the type system will help you ensure that all edge cases and combinations are covered. Otherwise, it's very easy write a bug where we forget to check the -1 case and end up with funky math (made especially dangerous by JS/TSs lack of a uint type!).

Reply:
It was -1 instead of 0 since teams with 0/0 issues should be sorted below those with 0/1. However, with the idea where we should sort by number of issues each team has received descending, that idea seems to be better here. I'll change this.